Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG PYTHON_VERSION=3.12.0
+FROM python:${PYTHON_VERSION}-slim as base
+LABEL maintainer="silva.vvss12@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /cinema_service
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
The `COPY . .` command copies all files from the current directory to the container. Ensure that your `.dockerignore` file is correctly set up to exclude unnecessary files.